### DISCUSSION TITLE
After updating @radix-ui/react-alert-dialog I started getting the error

### DISCUSSION THREAD

**Discussion by Mauladen** [radix-ui/primitives#2971] [Help]
After updating @radix-ui/react-alert-dialog from version 1.0.5 to the current one, I started getting an error

![PixelSnap 2024-06-22 at 21 32 37@2x](https://github.com/radix-ui/primitives/assets/12824589/b9018b6f-030e-465e-abfd-3e19a7840fba)

I don’t even know how to fix this because it’s not very clear where it’s happening

**Comment by manavm1990** [comment]
You might be using `{children}` somewhere? In that case, if you wrap it in a `<>` then it will be considered a single 🧒🏾 instead of multiple children.

**Reply by Mauladen** [reply]
> You might be using `{children}` somewhere? In that case, if you wrap it in a `<>` then it will be considered a single 🧒🏾 instead of multiple children.

Thank you for your answer, I understand that, but unfortunately I couldn't find this place, most of all I didn't like that I didn't find any information in the changelog to understand exactly where to look

**Reply by manavm1990** [reply]
🆗 If you can share more of your code and/or explain more about how you are implementing Radix `Dialog`...

**Comment by b0yblake** [comment]
Could you please share the full source code or a sample that replicates the error? It's hard to suggest a solution without it.
